flake.nix

```diff
@@ -194,10 +204,11 @@
           # purposes (like `nix build`).
           legacyPackages =
             let
-
-              overrideCargoProfileRecursively = deriv: profile: deriv.overrideAttrs (oldAttrs: {
```

Comment:
This was a terrible hack and now it's no more. Instead we use `overrideScope'` to override `commonProfile` which is used everywhere in `craneCommon.nix`.